source/examples/uiconcept/main.cpp

```diff
@@ -39,6 +40,14 @@ int main(int argc, char *argv[])
     // Load and show QML
     engine.load(QUrl::fromLocalFile(engine.qmlToolboxModulePath() + "/examples/uiconcept/window.qml"));
     
+    auto windows = app.allWindows();
+	if (!windows.empty())
+	{
+		auto window = windows.first();
```

Comment:
It shouldn't. It connects one window's signals (toFullScreen/toNormalScreen) to that same window's showFullScreen/showNormal. Only one window has the keyboard focus, so if the shortcut on that window, only that window should switch fullscreen mode.